archr/targets/docker_target.py

```diff
@@ -105,11 +105,10 @@ def build(self, pull=False):# pylint:disable=arguments-differ
         super().build()
         return self
 
-    def start(self, user=None, name=None, working_dir=None, labels=None, entry_point=None): #pylint:disable=arguments-differ
-        if labels is None:
-            labels = []
-        if entry_point is None:
-            entry_point = ["/bin/sh"]
+    def save(self, tag="latest"):
+        self.container.commit(f"{self.image_id}:{tag}")
```

Comment:
The `image_id` already has a tag associated with it if one was included in the `image_name` at construction. This would cause tags to look like `my_image:latest:latest` or similar. IMO the tag should be a regular required arg and be used directly, allowing the user flexibility.

Reply:
The new target_tag should be a required argument of the save or of the Target object constructor?

Reply:
Of the save method. Basically make it an alias to the `self.container.commit` method.